source.lua

```diff
@@ -1076,7 +1076,7 @@ end
 return {
 	bc_to_state = lua_bc_to_state,
 	wrap_state = lua_wrap_state,,
```

Comment:
Should probably remove the extra comma here and maybe rebase.

Reply:
Done